### PR TITLE
feat: add updated KinD example

### PR DESCRIPTION
**Pull request by tyrossel**
## Description of the changes

This PR creates the new KinD deployment example as well as adding the respective document page.

As a sidenote, we also updated the folder structure to better reflect the naming scheme used throughout the rest of the modules.

### CODE REVIEW

File: docs/modules/ROOT/pages/tutorials/deploy_kind.adoc [hunk 1]
```diff
@@ -78,6 +78,23 @@ This error happens when ArgoCD bootstraps itself.
...
+level=error ts=2023-05-09T06:32:38.495673778Z caller=main.go:117 msg="error creating promtail" error="failed to make file target manager: too many open files"
+Stream closed EOF for loki-stack/loki-stack-promtail-bxcmw (promtail)
+----
+
+You will have to increase the upper limit on the number of INotify instances that can be created per real user ID:
+
+[source,bash]
+----
+sudo echo '512' > /proc/sys/fs/inotify/max_user_instances
+----
```
Comment (lentidas) [inline docs/modules/ROOT/pages/tutorials/deploy_kind.adoc:96]: I found out that this command only applies temporarily and is reset on every session, at least that was my case. Alternatively, I think we can properly change the value with the command `sudo sysctl fs.inotify.max_user_instances=512`.

Comment (lentidas) [reply docs/modules/ROOT/pages/tutorials/deploy_kind.adoc:96]: Sorry, my bad! Actually this command also changes the value temporarily. If we want to have a permanent change, you'll need to edit the `/etc/sysctl.conf` (see [here](https://www.suse.com/support/kb/doc/?id=000020048) for reference).

Comment (tyrossel) [reply docs/modules/ROOT/pages/tutorials/deploy_kind.adoc:96]: We could propose both, if some users want to test the DevOps stack without changing their computer config ?

Comment (lentidas) [reply docs/modules/ROOT/pages/tutorials/deploy_kind.adoc:96]: That was what I was going to propose 😉

File: docs/modules/ROOT/pages/tutorials/deploy_kind.adoc [hunk 2]
```diff
@@ -1,3 +1,181 @@
...
+
+You will have to increase the upper limit on the number of INotify instances that can be created per real user ID:
+
+[source,bash]
+----
+# Increase the limit until next reboot:
+sudo sysctl fs.inotify.max_user_instances=512
+# Increase the limit permanently:
+sudo echo 'fs.inotify.max_user_instances=512' >> /etc/sysctl.conf
```
Comment (lentidas) [inline docs/modules/ROOT/pages/tutorials/deploy_kind.adoc:98]: I think we need to review this command, I don't think it works as expected.

File: docs/modules/ROOT/pages/tutorials/deploy_kind.adoc [hunk 3]
```diff
@@ -1,3 +1,181 @@
...
+$ terraform output keycloak_users
+{
+  "devopsadmin" = "aqhEbd3L0Msryjjp547ej7nyN6E2FllV"
+}
+----
+
+== Pause the cluster
+
+The `docker pause` command can be used to halt the cluster for a while in order
+to save energy:
```
Comment (lentidas) [inline docs/modules/ROOT/pages/tutorials/deploy_kind.adoc:144]:
```suggestion
The `docker pause` command can be used to halt the cluster for a while in order to save energy (replace `kind-cluster` by the cluster name you defined in `locals.tf`):
```

**Review by lentidas — CHANGES_REQUESTED** [review]

**Comment by lentidas** [comment]
We are nearing completion and waiting on the PRs from MinIO and Keycloak:
- https://github.com/camptocamp/devops-stack-module-keycloak/pull/16
- https://github.com/camptocamp/devops-stack-module-minio/pull/23

**Review by ckaenzig — APPROVED** [review]

**Pull request merged**